src/components/[guild]/Requirements/components/Requirement.tsx

```diff
@@ -35,76 +43,99 @@ const Requirement = ({
             }
-            alignItems="center"
-            justifyContent="center"
-            overflow={withImgBg ? "hidden" : undefined}
-          >
-            {typeof image === "string" ? (
-              image.endsWith(".mp4") ? (
-                <video
-                  src={image}
-                  width={"var(--chakra-space-11)"}
-                  height={"var(--chakra-space-11)"}
-                  muted
-                  autoPlay
-                  loop
+            isImageLoading={isImageLoading}
+            withImgBg={withImgBg}
+          />
+        </Box>
+        <VStack alignItems={"flex-start"} alignSelf="center" spacing={1.5}>
+          <HStack>
+            <Text wordBreak="break-word">
+              {requirement?.isNegated && <Tag mr="2">DON'T</Tag>}
+              {fieldRoot ? (
+                <RequirementNameEditor>
+                  {requirement?.data?.customName || children}
+                </RequirementNameEditor>
+              ) : (
+                requirement?.data?.customName || children
+              )}
+
+              {fieldRoot ? (
+                <SetVisibility
+                  ml={2}
+                  entityType="requirement"
+                  fieldBase={fieldRoot}
                 />
               ) : (
-                <Img
-                  src={image}
-                  maxWidth={"var(--chakra-space-11)"}
-                  maxHeight={"var(--chakra-space-11)"}
+                <Visibility
+                  entityVisibility={requirement?.visibility ?? VisibilityType.PUBLIC}
+                  ml="1"
                 />
-              )
-            ) : (
-              image
-            )}
-          </Circle>
-        </SkeletonCircle>
-      </Box>
-      <VStack alignItems={"flex-start"} alignSelf="center">
-        <HStack>
-          <Text wordBreak="break-word">
-            {requirement?.isNegated && <Tag mr="2">DON'T</Tag>}
-            {children}
-            {fieldRoot ? (
-              <SetVisibility ml={2} entityType="requirement" fieldBase={fieldRoot} />
-            ) : (
-              <Visibility
-                entityVisibility={requirement?.visibility ?? VisibilityType.PUBLIC}
-                ml="1"
-              />
-            )}
-          </Text>
-        </HStack>
+              )}
+            </Text>
+          </HStack>
 
-        {footer}
-      </VStack>
-      {rightElement}
-    </SimpleGrid>
+          <HStack>
+            {previewAvailable && !isEditing && (
+              <Popover placement="bottom-start">
+                <PopoverTrigger>
+                  <RequirementButton
+                    rightIcon={<Icon as={CaretDown} />}
+                    onClick={togglePreview}
+                  >
+                    View original
+                  </RequirementButton>
+                </PopoverTrigger>
+                <Portal>
+                  <PopoverContent minW="max-content">
+                    <OriginalRequirementPreview
```

Comment:
I think we could completely get rid of this component and use the `RequirementDisplayComponent` instead with `footer={null}` and `rightElement={!!fieldRoot ? <ResetRequirementButton /> : null}`